index.rst

```diff
@@ -1,57 +1,332 @@
+This evolves further as one considers what tooling and knowledge is required for development of the system, however, all of the above users and usages require those interactive tools.
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+The breadth of the observatory control system allows users to perform both these routine and customized tasks via three main interactive components: the :ref:`LSST Observing Visualization Environment (LOVE)<LOVE>`, :ref:`ScriptQueue <Script-Queue>` and :ref:`Jupyter notebooks <Jupyter-notebooks>`.
+This document begins with a high-level overview of the systems and explore how users can take advantage of their features to accomplish different types of tasks
+References to more detailed documentation regarding their operation are linked throughout..
 
-   Feel free to delete this instructional comment.
+Then, to assist in defining the expectations and knowledge levels associated with the multiple complexities associated with all of the required tasks that must be performed by the OCS, we've identified three main actors and their expected roles and ability levels associated their interactions with the control system.
+The identified actors are: 1) Observing Specialists, 2) Commissioning Scientists, 3) Software Developers.
+In reality, we expect (and need) several people to have significant overlap between these roles.
 
-:tocdepth: 1
+The documentation associated with the example procedures expected to be performed by each of the actors are then linked to provide a demonstration of the current usability level associated with each task.
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+Under the standard operational procedures used to perform the survey, the observatory will be largely driven by the Scheduler, which will designate a visit script, which consists of a series of operations that are performed via a content controlled script. The script will be executed by the scriptQueue, and the user will be able to observe (or interrupt) the progress via the LOVE interface.
+It is expected that operators will be much more involved during the start-up and shutdown phases, as well as during afternoon calibrations.
+In the majority of cases, the operations will be routine and are expected to be done from a high-level of the observatory control software, as is discussed further in following sections.
 
-.. sectnum::
+Lastly, using concrete examples, we present examples of the types of interactions that these actors will perform with links to procedures on how to do them using the multiple interfaces.
+These examples explore how users can take advantage of the features also include cases of how *not* to use an interface due to its limitations or inefficiencies in some areas.
```

Comment:
Uhm... this phrase is a bit confusing. Can you rephrase it?

Reply:
rephrased